receiver/redisreceiver/testdata/config.yaml

```diff
@@ -0,0 +1 @@
+# TODO: create issue
```

Comment:
This is already resolved https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/11075, please rebase